nuget.config

```diff
@@ -3,6 +3,6 @@
   <packageSources>
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" />
-    <!--<add key="Local" value="c:\nuget" />-->
+    <add key="Local" value="C:\src\nunit3-vs-adapter\package" />
```

Comment:
Should not be kept, this is to allow working with the samples.

Reply:
Should not be here:  
```suggestion
    <!--<add key="Local" value="C:\src\nunit3-vs-adapter\package" /> -->
```

Reply:
Add a file  `nuget.config` in the samples directory:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
    <add key="Local" value="../package" />
  </packageSources>
</configuration>

```